sms_sponsorship/models/sms_child_request.py

```diff
@@ -66,7 +66,7 @@ class SmsChildRequest(models.Model):
 
     new_partner = fields.Boolean('New partner ?',
                                  help="is true if partner was created when "
-                                 "sending sms", default=True)
+                                 "sending sms", default=False)
```

Comment:
If this fixes the bug, you should have a commit named CO-1965.

Reply:
Ok, on this pul, request or on a new one ?

Reply:
On this one directly. I'm waiting for your fixes.

Reply:
You can amend commit to edit the message just in case.